pkg/domain/resourcegroup/runaway.go

```diff
@@ -476,7 +476,7 @@ type RunawayChecker struct {
 	watchAction   rmpb.RunawayAction
 }
 
-func newRunawayChecker(manager *RunawayManager, resourceGroupName string, setting *rmpb.RunawaySettings, originalSQL, sqlDigest, planDigest string) *RunawayChecker {
+func newRunawayChecker(manager *RunawayManager, resourceGroupName string, setting *rmpb.RunawaySettings, originalSQL, sqlDigest, planDigest string, startTime time.Time) *RunawayChecker {
```

Comment:
seems some params passed from `sessionVars`, maybe directly pass the sessVars in here?

Reply:
After testing, it appears that moving `GetPlanDigest` causes an import cycle. Therefore, I suggest we keep it as is for now.